File: Entities/Entity.py


class Entity:
    entity_id = None
    name = None
    description = None
    linked_entities = []
    
    def __init__(self, entity_id, name, description, linked_entities):
        self.entity_id = entity_id
        self.name = name
        self.description = description
        self.linked_entities += linked_entities
    
    def getEntityID(self):
        return self.entity_id

File: src/main/java/de/jj22/telegram_bot_rpg/Game/Entities/Entity.java
package de.jj22.telegram_bot_rpg.Game.Entities;

public class Entity {
	final private int entity_id;
	final private String name;
	final private String description;

	public Entity(int entity_id, String name, String description) {
		this.entity_id = entity_id;
		this.name = name;
		this.description = description;
	}

	public int getEntityID() {
		return this.entity_id;
	}
	
	public String getDescription() {
		return description;
	}
	
	public String getName() {
		return name;
	}
	
}
